design_docs/openapi.yml

```diff
@@ -6,166 +6,229 @@ info:
     name: TBD
     url: TBD
 servers:
-  - url: http://TBD.mongodb.com/v1/ # URL TBD
+  - url: http://TBD.mongodb.com/v1
+security:
+  - ApiKeyAuth: []
 paths:
-  /start:
+  /conversations:
     post:
```

Comment:
this is now missing the request body. pls add back

Reply:
or do you want to get IP address through different means? HTTP header?

Reply:
We can use express middleware to pull the IP address from every incoming request - no need to manually specify in a request body.